SU-SWS/saa_victoria_subtheme
# Adapt 290 basic theme setup, file organization, tailwind configuration

# READY FOR REVIEW

# Summary
- Adding scss and theme organization, basic setup, integrating tailwind with custom settings

# Review By (Date)
- Friday may 22nd if possible before the long weekend, i will continue my  next work off a branch from this branch

# Urgency
- low

# Steps to Test

1. You can see that the @apply class lines in scss/theme/_button.scss apply tailwind styles if you compile 
2. check theme.css to see the reflected color styles correctly coming in from tailwind classes using our decanter colors


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA tickets: ADAPT-290 for basic theme setup, https://stanfordits.atlassian.net/browse/ADAPT-306 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
